static/css/style.css

```
@@ -1,5 +1,1490 @@
-.test {
-    background-color: gold;
-    border: 100px;
+* {
+  margin: 0;
+  box-sizing: border-box;
+  z-index: 1; }
+
+body {
+  margin: 0 0 1px 0;
+  background: linear-gradient(253.81deg, #59BA69 30.24%, #ECFB92 86.74%); }
+
+.container {
+  padding: 0 100px; }
+  @media (max-width: 706px) {
+    .container {
+      padding: 0 20px; } }
+  @media (max-width: 420px) {
+    .container {
+      padding: 0 10px; } }
+
+.flex-break-item {
+  width: 100%; }
+  .flex-break-item_footer {
+    width: unset; }
+    @media (max-width: 920px) {
+      .flex-break-item_footer {
+        width: 100%; } }
+
+html {
+  overflow-x: hidden; }
+
+input[type="checkbox"]:checked,
+input[type="checkbox"]:not(:checked),
+input[type="radio"]:checked,
+input[type="radio"]:not(:checked)
+ {
+  position: absolute;
+  left: -9999px; }
```

???

Чаму проста не

```suggestion
input[type="checkbox"],
input[type="radio"]
 {
  position: absolute;
  left: -9999px;
  visibility: hidden;
}
```